.babelrc

```diff
@@ -1,5 +1,5 @@
 {
-    "presets": ["stage-0", "react"],
+    "presets": ["stage-0", "react", "es2015"],
```

Comment:
The `es2015` was removed here on purpose to support webpack tree-shaking. (I agree however that we might need to have the other transforms performed).
```json
"presets": ["stage-0", "react", ["es2015", { "modules": false }]],
```

Branch https://github.com/dhis2/d2-ui/tree/v25.0.x still contains es2015, but it would need the cyclic dependency fix you mentioned.